src/packages/cli/src/Generate.ts

```diff
@@ -153,7 +186,7 @@ export class Generate implements Command {
           }
         }
       })
-      await new Promise(r => null)
+      await new Promise((_) => null) // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
I think you can remove the no-unused-vars here since you have an underscore

Reply:
When removing the comment I still get `189:26  warning  '_' is defined but never used  @typescript-eslint/no-unused-vars` so I'll keep the comment I guess :)

Reply:
You can add a rule to ignore variables starting with a _